Menu/Menu.jsx

```diff
@@ -1,46 +1,48 @@
 import './Menu.styl'
 
-export default class Menu extends React.Component {
-  static propTypes = {
-    closeMenu: PropTypes.func.isRequired
+if (config.is_preact) {
```

Comment:
This `try..catch` block works with both modules: `react` **and** `preact`?

Reply:
This updated try..catch block works with react , preact and vanill js